source/transactions/transactions.rst

```diff
@@ -150,6 +150,22 @@ This section is an overview of the public API for transactions:
          * The readPreference to use for this transaction.
          */
         Optional<ReadPreference> readPreference;
+
+        /**
+         * The readPreference to use for this transaction.
+         */
+        Optional<ReadPreference> readPreference;
+
+        /**
+         * The maximum amount of time to allow the *commit* to run.
+         *
+         * This option is only sent with the commitTransaction command and
+         * only if the caller explicitly provides a value.
+         * The default is to not send a value.
+         *
+         * @see https://docs.mongodb.com/manual/reference/command/commitTransaction/
+         */
+        Optional<Int64> maxTimeMS;
```

Comment:
Would it make sense to copy in some text elsewhere in the spec about permitted naming deviations, as we do for `maxTimeMS` in the [CRUD spec](https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#id1)? That's assuming that some drivers might use an alternate type for this option and forgo the "MS" suffix.

Reply:
Good idea, done.